src/domain/shared/Uuid.ts

```diff
@@ -0,0 +1,69 @@
+//
```

Comment:
Why did you implement your own UUID implementation rather than using the UUID npm package? The library package would make sure it followed the standard.

Reply:
I can't recall whether or not I looked for a Uuid package. Perhaps an NPM package might be useful in the future. However, at present, we only use UUID values in their raw binary format, e.g., as received from the domain server.

Reply:
If this class is only for managing the binary blob from the server, then maybe use another name so that, in the future, it doesn't conflict with classes meant manipulating UUIDs. "BinUUID", "RawUUID", "ServerID", ...
Not a biggie and not worth the time fiddling at the moment.

Reply:
I imagine that we'll switch to using the npm Uuid package sometime in the future if and when we need to do more with Uuids.

Reply:
I already use it in the vircadia-web repository. This function can always be fixed later, though.